product-service/libs/services/products.service.ts

```diff
@@ -0,0 +1,37 @@
+import products from "../../products.json";
+
+interface ProductInterface {
+    id?: string,
+    title: string,
+    description: string,
+    price: number,
+    logo: string,
+    count?: number
+}
+
+interface ProductsServiceInterface {
+    getProductById( id: string ): undefined | ProductInterface
+    getAllProducts(): ProductInterface[]
+}
+
+class ProductsService implements ProductsServiceInterface {
+    getProductById( id: string ) {
+        try {
+            return products.find( product => product.id === id );
+        }
+        catch( error ) {
+            console.log( `Method ProductsModel::getProductById. Error: ${ error }` );
```

Comment:
( `There is no product with id: ${ id }` );

Logs for support not for dev

Reply:
Added
![image](https://user-images.githubusercontent.com/36921389/113278000-07745880-92ea-11eb-994c-b44d1c79f127.png)

![image](https://user-images.githubusercontent.com/36921389/113278025-10652a00-92ea-11eb-8353-8c8ba1b0e3ce.png)

Reply:
Now logger is working by next logic: if we are on the PROD - log only errors level messages. In others stages - info level
![image](https://user-images.githubusercontent.com/36921389/113278178-41ddf580-92ea-11eb-9a12-20b96cfd5944.png)